src/test/java/net/snowflake/client/providers/SimpleFormatProvider.java

```diff
@@ -0,0 +1,13 @@
+package net.snowflake.client.providers;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+
+public class SimpleFormatProvider extends SnowflakeArgumentsProvider {
```

Comment:
I thought that it simple format for data time :) - can we rename this class to `SimpleResultFormatProvider`?